src/main/java/com/dreamhorizon/enchantments/objects/DHEnchantment.java

```diff
@@ -103,4 +109,12 @@ public boolean canEnchantItem(@NotNull ItemStack item) {
         }
         return enchantableItems.contains(item.getType());
     }
+
+    public String getGoesOn() {
```

Comment:
Maybe getApplicable() but also not something necessary "goes on" just sounds a bit dull.